docgen/src/Customization.md

```diff
@@ -10,10 +10,10 @@ While react-instantsearch already provides widgets out of the box, there are cas
 
 All default widgets have a corresponding higher-order component that acts as a connector, providing the required props to the widget.
 
-Those connectors are available as the `connect` property on the widgets they connect. For instance, the `SearchBox` widget uses the `SearchBox.connect` higher-order component under the hood in order to retrieve and refine the current query.
+Those connectors are available via `import connectName from from 'react-instantsearch/connectors`. For instance, the `SearchBox` widget uses the `connectSearchBox` higher-order component under the hood in order to retrieve and refine the current query.
```

Comment:
I will rephrase it and do an example wit SearchBox, this should make it obvious. Thanks